src/taco-cli/cli/utils/cordovaHelper.ts

```diff
@@ -315,6 +318,23 @@ class CordovaHelper {
             });
     }
 
+    public static getCordovaExecutable(): Q.Promise<string> {
+        return projectHelper.getProjectInfo()
+            .then(function (projectInfo: projectHelper.IProjectInfo): any {
```

Comment:
This can return a `string | Q.Promise<string>` rather than `any`, to be a little more precise.